### PR TITLE
Update free-courses-hi.md

### PR DESCRIPTION
**Pull request by decentlearner**
added resource for graph playlist

## What does this PR do?
Add resource(s) | Remove resource(s) | Add info | Improve repo

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: courses/free-courses-hi.md [hunk 1]
```diff
@@ -85,6 +85,7 @@
 * [DSA-One Course- The Complete Data Structure and Algorithms Course](https://www.youtube.com/playlist?list=PLUcsbZa0qzu3yNzzAxgvSgRobdUUJvz7p) - Anuj Bhaiya (YouTube)
 * [Dynamic Programming Playlist](https://www.youtube.com/playlist?list=PL_z_8CaSLPWekqhdCPmFohncHwz8TY2Go) - Aditya Verma
 * [Functions and Arrays - Level 1](https://www.youtube.com/playlist?list=PL-Jc9J83PIiHOV7lm2uSw4ZiVsIRsGS6r) - Pepcoding
+* [Graph Playlist\| Interview Questions \| Coding \| Tutorials \| Data Structures](https://www.youtube/playlist?list=PLgUwDviBIf0oE3gA41TKO2H5bHpPd7fzn) - takeUforward
```
Comment (davorpa) [inline courses/free-courses-hi.md:88]: - Fixing url
- Using title as it is
- Add instructor as part of the creators

```suggestion
* [Graph Series by Striver \| C++ \| Java \| Interview Centric \| Algorithms \| Problems](https://www.youtube.com/playlist?list=PLgUwDviBIf0oE3gA41TKO2H5bHpPd7fzn) - take U forward, `tch.:` Raj Striver
```

**Review by davorpa — CHANGES_REQUESTED** [review]

**Comment by decentlearner** [comment]
Changes made as suggested

**Review by davorpa — APPROVED** [review]

**Comment by decentlearner** [comment]
Please accept and add label hacktoberfest-accepted if this fulfill all changes suggested.

**Comment by davorpa** [comment]
> Please accept and add label hacktoberfest-accepted if this fulfill all changes suggested.

Not needed since repo is marked as hacktoberfest. Will count when merged

**Comment by eshellman** [comment]
could we NOT add tch. here? it's implied by the genre, and probably confusing to most.

File: courses/free-courses-hi.md [hunk 2]
```diff
@@ -85,6 +85,7 @@
 * [DSA-One Course- The Complete Data Structure and Algorithms Course](https://www.youtube.com/playlist?list=PLUcsbZa0qzu3yNzzAxgvSgRobdUUJvz7p) - Anuj Bhaiya (YouTube)
 * [Dynamic Programming Playlist](https://www.youtube.com/playlist?list=PL_z_8CaSLPWekqhdCPmFohncHwz8TY2Go) - Aditya Verma
 * [Functions and Arrays - Level 1](https://www.youtube.com/playlist?list=PL-Jc9J83PIiHOV7lm2uSw4ZiVsIRsGS6r) - Pepcoding
+* [Graph Series by Striver \| C++ \| Java \| Interview Centric \| Algorithms \| Problems](https://www.youtube.com/playlist?list=PLgUwDviBIf0oE3gA41TKO2H5bHpPd7fzn) - take U forward, `tch.:` Raj Striver
```
Comment (davorpa) [inline courses/free-courses-hi.md:88]:
```suggestion
* [Graph Series by Striver \| C++ \| Java \| Interview Centric \| Algorithms \| Problems](https://www.youtube.com/playlist?list=PLgUwDviBIf0oE3gA41TKO2H5bHpPd7fzn) - take U forward
```

**Pull request merged**